### DISCUSSION TITLE
[Feature Request]: 词典-自定义识别词增加副标题

### DISCUSSION THREAD

**Discussion by Z2112** [jxxghp/MoviePilot#3465] [讨论]
### 当前程序版本

v1.7.9

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

“自定义识别词”只是生效在主标题里，副标题不生效导致的屏蔽失败。
主标题是
“Tensei shitara Slime Datta Ken S03 1080p Crunchyroll WEB-DL H264 AAC”
副标题是
“[2024年4月新番] 关于我转生变成史莱姆这档事 第三季 / 転生したらスライムだった件 第3期 第01集 内封中字”
目前的识别结果是S03E03，是因为副标题里的“第3期”被识别了03集。
希望能增加一个副标题也能修改的地方。

**Comment by chenvd** [comment]
特别是某些站点主标题是不包含集数信息，然后副标题分季和TMDB是不一样，需要做偏移的时候
